### ISSUE TITLE
render asciidoc files too

### ISSUE THREAD

**Issue by gernotstarke** [joeyespo/grip#92]
enhance grip to render both Markdown and [Asciidoc](http://asciidoctor.org) files.

Why that:
Asciidoc currently becoming more and more popular...

**Comment by joeyespo** [comment]
I'm all for supporting the best formats with high adoption. The problem with integrating this into Grip is that it [currently uses](https://github.com/joeyespo/grip#access) GitHub's [Markdown API](https://developer.github.com/v3/markdown/) to render markdown files exactly like GitHub would.

While this *could* be a candidate for the offline rendering mode, I'm not convinced that it should offer more features than Grip's normal usage.

I'd be happy to add more formats if GitHub adds an endpoint for them. If Asciidoc becomes as popular as you're suggesting, GitHub may one day use it as their site-wide default in place of Markdown (or offer it in parallel somehow.) This would result in a dedicated GitHub-Flavored Asciidoc endpoint Grip could use. Until something like that happens, I'm going to mark this as out-of-scope to keep Grip focused.

Feel free to clone grip for a Asciidoc-only solution. (Call it grap? :-) Most likely, you'll then remove the normal API mode and use asset-downloading and offline rendering exclusively.

Thanks for the request.
